docs/src/pages/[platform]/components/expander/examples/CustomTitleComponent.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+
+import {
+  Expander,
+  ExpanderItem,
+  Flex,
+  View,
+  useTheme,
+} from '@aws-amplify/ui-react';
+
+export const CustomTitle = ({ courseNumber, courseName }) => {
+  const { tokens } = useTheme();
+  return (
+    <Flex gap={tokens.space.small}>
+      <View width={tokens.space.xxxl} color={tokens.colors.teal[80]}>
+        {courseNumber}
+      </View>
+      <View>{courseName}</View>
+    </Flex>
+  );
+};
+
+export const CustomTitleComponent = () => {
+  return (
+    <Expander type="single">
+      <ExpanderItem
+        title={
+          <CustomTitle
+            courseNumber="CS 103"
+            courseName="Mathematical Foundations of Computing"
+          />
```

Comment:
Love the real world example here! And good clean React code making a separate component!